src/hooks/useParamsForm.ts

```diff
@@ -0,0 +1,29 @@
+import { ChangeEvent, useCallback } from 'react'
+import { useSelector } from 'react-redux'
+import useDispatch from './useDispatch'
+import { getSelectedComponentId } from '~core/selectors/components'
+
+export const useParamsForm = () => {
+  const dispatch = useDispatch()
+  const componentId = useSelector(getSelectedComponentId)
+
+  const setValueFromEvent = ({
```

Comment:
where is it needed?

Reply:
> where is it needed?

Removed! - [link](https://github.com/trilogy-group/tu2k22-openchakra/pull/2/commits/181e1dfe359f8165b2f806616491d18299a84574)